sdk/core/azure-core-xml/vcpkg/vcpkg.json

```
@@ -0,0 +1,36 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+{
+  "$comment": [
+    "NOTE: All changes made to this file will get overwritten by the next port release.",
+    "Please contribute your changes to https://github.com/Azure/azure-sdk-for-cpp."
+  ],
+  "name": "azure-core-xml-cpp",
+  "version-semver": "@AZ_LIBRARY_VERSION@",
+  "description": [
+    "Azure Core Xml provides an implementation of serializing/deserializing Xml documents in C++."
```

```suggestion
    "This library provides support for modern Azure SDK client libraries written in C++ to leverage XML parsing."
```

(wording is taken from `azure-core-tracing-opentelemetry-cpp`, which is also an internal library).